mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/gridfs/GridFSTest.java

```diff
@@ -153,6 +164,8 @@ public void canRemoveEntityWithBlob() {
 			assertBlobAreEqual( photo.getContentAsBlob(), BLOB_CONTENT_1 );
 			assertThat( photo.getContentAsByteArray() ).isNotNull();
 			assertThat( photo.getContentAsByteArray() ).isEqualTo( BLOB_CONTENT_1 );
+			assertThat( photo.getContentAsString() ).isNotNull();
+			assertThat( photo.getContentAsString() ).isEqualTo( BLOB_STRING_CONTENT_1 );
```

Comment:
I don't think these tests are right for this issue. They would pass even if you don't store the field using GridFS. Is there a way to check if the field has been store effectively using GridFS?

Also there is no need to asser

Reply:
you right.  About your question. Yes. We can check GridFS. We can write additional information to meta.

Reply:
Yes, I guess... you need to make sure that it contains the values you expect. Does that asnwer your question?

Reply:
Hi @DavideD !

Yes,of course.